For some reason the relative path to the test data doesn't work in our
build environment, but using the absolute path passed by CMake via
the -DTEST_DATADIR flag fixes that.

Need to investigate to determine if this is our fault or should be fixed
upstream.

--- libical-1.0.1/src/test/recur.c
+++ libical-1.0.1/src/test/recur.c
@@ -82,7 +82,7 @@
 #endif
 	
     if (argc <= 1){
-		file = "../../test-data/recur.txt";
+		file = TEST_DATADIR "/recur.txt";
     } else if (argc == 2){
 		file = argv[1];
     } else {

--- libical-1.0.1/src/test/testvcal.c
+++ libical-1.0.1/src/test/testvcal.c
@@ -44,7 +44,7 @@
     char* file;
 
     if (argc != 2){
-        file = "../../test-data/user-cal.vcf";
+        file = TEST_DATADIR "/user-cal.vcf";
     } else {
         file = argv[1];
     }
